src/stores/login.ts

```diff
@@ -31,6 +40,11 @@ interface User {
   last_name: string;
 }
 
+interface RefreshTokenResponse {
+  access: string;
+  access_token_expiration: string;
```

Comment:
`access_token_expiration` key should be ` access_expiration`.

Reply:
Updated.